prov/tcp/src/xnet_pep.c

```diff
@@ -241,8 +252,14 @@ int xnet_listen(struct xnet_pep *pep, struct xnet_progress *progress)
 	}
 
 	ofi_genlock_lock(&progress->lock);
-	ret = xnet_monitor_sock(progress, pep->sock, POLLIN,
-				&pep->util_pep.pep_fid.fid);
+	if (xnet_io_uring) {
+		ret = xnet_uring_poll_add(progress, pep->sock, POLLIN,
+					  true, &pep->pollin_sockctx, NULL);
```

Comment:
If I understand correctly, multishot means that this pollin_sockctx will remain inuse even after an event has been written the cqe.  When is that cleared?